### DISCUSSION TITLE
How to make it delete immediately?

### DISCUSSION THREAD

**Discussion by lsshawn** [folke/flash.nvim#117] [Q&A]
When I delete word, Flash highlights the word that will be deleted but doesn't delete it until the next keypress.

How can I configure it to delete immediately just like regular Vim?

GIF below:

![Peek 2023-07-03 00-16](https://github.com/folke/flash.nvim/assets/28279366/8fab4bad-0140-49f1-824d-a59b1303236a)

**Comment by folke** [comment]
please update, I accidentally made labels for char motions the default, which was not intentional
